APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,212 @@
+difficult to design a development and review process that works effectively with
+*both* of these classes of packages.
+
+Second, the current review process has become somewhat opaque and rather awkward
+to keep organized.  The coordination committee reviews the packages, which in
+practice nearly always receive a "provisional" designation, but then must be
+re-reviewed as a *whole* at some future date to become "full" affiliated
+packages. This "provisional" and "full" distinction is opaque and probably
+meaningless to the average user who just wants to know what a package does, and
+if it is any good. Additionally, while the review standards are stated on the
+affiliated package web page, some are subjective, allowing at least the
+possibility of biased or exclusionary behavior to sneak in with little recourse
+given the blunt nature of "reject", "provisional", and "accept".
+
+Third, the coordination committee has found it difficult to keep up with the
+rate of affiliated package review (and re-review) requests. Many reviews
+languish for months before receiving even a "provisional" response, which can be
+disheartening and perceived by some as unwelcoming. This is primarily driven by
+the need to do a detailed and sometimes lengthy review of each package, even if
+that amounts to just following a checklist (see the section on the new review
+process for more details of the current process in practice).
+
+These issues were discussed at length by the coordination committee, and later
+in the 2017 Astropy coordination meeting.  The new structure and processes
+described below were a product of a breakout session at the coordination
+meeting. This APE is thus an effort to formalize the ideas and describe how to
+implement them.
+
+Astropy "coordinated" and affiliated packages
+=============================================
+
+The biggest-picture change is to split the currently single-category affiliated
+package ecosystem into two categories: "Astropy coordinated" and "Astropy
+affiliated" package.  This change is not as drastic as it sounds because it
+reflects the *reality* of the current situation.  As outlined above, two such
+categories effectively exist already, and this APE simply formalizes this
+status.
+
+In practice, packages that remain as affiliated packages are essentially
+unaffected by this change.  They continue to be managed/organized outside of the
+Astropy project core team, and retain complete control of the repositories, code
+base, etc.  By contrast, packages in the new "coordinated package" category will
+be treated as a core component of the Astropy project, with the project (via the
+coordinating committee) overseeing who is maintaining the package and taking
+responsibility for continued maintenance in the event the original author(s) no
+longer can do so. This means maintainer roles will be created for all the
+coordinated packages, and have full status as astropy
```

Comment:
remove 'full' here (again, a nitpick, but it implies there might be a 'partial' astropy role... I don't think there is?)